src/reports/client/main.cpp

```diff
@@ -0,0 +1,199 @@
+#include <chrono>
+#include <cmath>
+#include <iostream>
+#include <thread>
+#include <vector>
+
+#ifdef HAVE_MPI
+#include <mpi.h>
+#endif
+
+#include <bbp/sonata/reports.h>
+#include <reports/utils/logger.hpp>
+
+struct Neuron {
+    int node_id;
+    std::string kind;  // soma / element
+    std::vector<double> voltages;
+};
+
+const static double dt = 0.1;
+const static double tstart = 0.0;
+const static double tstop = 0.3;
```

Comment:
They are used in several functions. I guess I could pass them around, but since is a test client I thought it was not so bad to keep them as global.

Ideally in the future the client could be parametrized to test different input values.